identity-account-storage/Cargo.toml

```diff
@@ -48,7 +51,7 @@ rusty-fork = { version = "0.3" }
 tokio = { version = "1.17.0", default-features = false, features = ["macros", "rt", "rt-multi-thread", "sync"] }
 
 [features]
-default = ["stronghold", "send-sync-storage"]
+default = ["stronghold", "send-sync-storage", "storage-test-suite"]
```

Comment:
I agree. I only added this because `cargo test` without `--all-features` will fail otherwise (e.g. in CI). I assume it's fine to change CI in that case? Considering we already run `cargo check/clippy --all-features` it should be fine, even though we end up testing fewer permutations of features as a slight downside.

Reply:
Ideally we would be testing three times: with `--all-features`, with default features, and with `--no-default-features`. The only problem is that causes three compilations that cannot be cached properly.

I would be fine with just adding `--all-features` to the CI, under the`.github/workflows/build-and-test.yml` workflow.

Reply:
I would also be fine with just running `--all-features` in the CI and encourage contributors to run `cargo hack check --feature-powerset --no-dev-deps` locally (unless that turns out to take way too long) (see [cargo-hack](https://github.com/taiki-e/cargo-hack#--feature-powerset)).